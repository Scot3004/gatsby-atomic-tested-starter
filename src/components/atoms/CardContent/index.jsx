import styled from 'styled-components';

export const CardContent = styled.div`
  flex: 1;
`;

export default CardContent;
