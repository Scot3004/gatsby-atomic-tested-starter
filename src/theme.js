const theme = {
  backgroundColor: '#f5f5f5',
  textColor: '#000',
  mutedColor: '#555',
  primaryColor: '#663399',
  visitedColor: '#402060'
};

export default theme;
