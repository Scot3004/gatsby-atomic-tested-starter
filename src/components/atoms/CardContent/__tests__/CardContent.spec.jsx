import React from 'react';
import 'jest-styled-components';
import renderer from 'react-test-renderer';
import CardContent from '..';
import theme from '../../../../theme';

describe('When I want to see a card element', () => {
  const tree = renderer.create(<CardContent theme={theme} />).toJSON();
  test('Then the content should not exceed the card dimensions', () => {
    expect(tree).toMatchSnapshot();
  });
});
