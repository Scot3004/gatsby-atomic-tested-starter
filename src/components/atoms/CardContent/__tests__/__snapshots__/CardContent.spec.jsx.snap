// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When I want to see a card element Then the content should not exceed the card dimensions 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-hyphens: auto;
  -moz-hyphens: auto;
  -ms-hyphens: auto;
  hyphens: auto;
  min-width: 0;
  overflow-wrap: break-word;
  word-wrap: break-word;
}

<div
  className="c0"
/>
`;
